subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/MarkReportAsSourceForTMSStep.kt

```
@@ -0,0 +1,40 @@
+package com.avito.ci.steps
+
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import com.avito.impact.configuration.internalModule
+import com.avito.instrumentation.instrumentationTask
+import com.avito.plugin.cthulhuPluginId
+import com.avito.plugin.markReportAsSourceTask
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.tasks.TaskProvider
+
+class MarkReportAsSourceForTMSStep(context: String, name: String) : BuildStep(context, name),
+    ImpactAnalysisAwareBuildStep by ImpactAnalysisAwareBuildStep.Impl() {
+
+    var configuration: String = ""
```

add fail fast on empty

---

thanks, will do